src/plugins/navigation_embeddable/public/components/dashboard_link/dashboard_link_component.tsx

```diff
@@ -96,37 +147,25 @@ export const DashboardLinkComponent = ({
     <EuiListGroupItem
       size="s"
       color="text"
-      isDisabled={Boolean(error)}
+      {...onClickProps}
       id={`dashboardLink--${link.id}`}
+      showToolTip={Boolean(error)}
```

Comment:
This needs to always be `true` so that the `EuiTooltip` overrides the browser tooltip even when not in an error state.

```suggestion
      showToolTip={true}
```